common/defaults.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 The go-ethereum Authors
```

Comment:
Well, I can agree, but the DefaultDataDir and DefaultIpcSocket were always part of common. Soo, along those lines, the rest of the RPC pieces should also go here :D 

Also it's not so easy to remove these, because before my PR, these values here were hard coded at a lot of places into the code separately. Moving them into `cmd/geth` would make them unimportable from other packages. Still, I'm open for any suggestion on how to do this better.